test/test_t1.py

```diff
@@ -0,0 +1,216 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+"""
+Test T1 experiment
+"""
+
+import unittest
+import numpy as np
+from qiskit.providers import BaseBackend
+from qiskit.providers.models import BackendConfiguration
+from qiskit.result import Result
+from qiskit_experiments import ExperimentData
+from qiskit_experiments.composite import ParallelExperiment
+from qiskit_experiments.characterization import T1Experiment, T1Analysis
+
+
+class T1Backend(BaseBackend):
+    """
+    A simple and primitive backend, to be run by the T1 tests
+    """
+
+    def __init__(self, t1, initial_prob1=None, readout0to1=None, readout1to0=None):
+        """
+        Initialize the T1 backend
+        """
+
+        configuration = BackendConfiguration(
+            "t1_simulator",
+            "0",
+            int(1e6),
+            ["barrier", "x", "delay", "measure"],
+            [],
+            True,
+            True,
+            False,
+            False,
+            False,
+            int(1e6),
+            None,
```

Comment:
Add comments to the parameters so that the user knows their meaning (beyond true/false)

Reply:
done